CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## IN PROGRESS
 
+* Prompt for unsaved changes when clicking on the Lists app home from the app context menu [UILISTS-177]
```

Comment:
Don't forget to add the `[UILISTS-177]: https://folio-org.atlassian.net/browse/UILISTS-177` to the file somewhere, too, so that this `[UILISTS-177]` link works 🙂

Reply:
Done!